.circleci/config.yml

```diff
@@ -27,11 +27,11 @@ jobs:
           when: always
           name: 'Integration Tests Setup'
           command: |
-            aws s3 cp s3://com-stitchdata-dev-deployment-assets/environments/tap-tester/sandbox dev_env.sh
+            aws s3 cp s3://com-stitchdata-dev-deployment-assets/environments/tap-tester/tap_tester_sandbox dev_env.sh
       - run:
```

Comment:
@harshpatel4crest Please add unit tests step in the CircleCI

Reply:
Added